src/components/Banner/__snapshots__/Banner.spec.jsx.snap

```diff
@@ -1,1278 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Blergh.  The process used to be that deleting the snapshots caused new ones to be created.  What's the process now?  I don't see anything in CONTRIBUTING or README.